backend/templateRecs/templateRecommendation.js

```diff
@@ -0,0 +1,67 @@
+import { calculateBaseScore } from "../controllers/recommendationController.js";
+import { prisma } from "../utils/helpers.js";
+
+const calculateLocality = async (userId, creatorId) => {
+  const maxDepth = 2;
+  const visited = new Set();
+  const queue = [[userId, 0]];
+  let mutualFollowers = 0;
+
+  // Breadth first search approach to finding second degree connections
+  while (queue.length > 0) {
+    const [currentUserId, depth] = queue.shift();
+
+    if (depth > maxDepth) break;
```

Comment:
nit: in general, it's good practice to always include brackets for if/else statements. While excluding them seems to make it more concise, it can lead to a lot of unexpected issues for other folks touching the code

Reply:
Thanks for the catch! I'll make sure to include the brackets